Server/Settings.cs

```diff
@@ -59,6 +59,9 @@ internal static class Settings
                              RelogDelay = 50,
                              MaxIP = 5;
 
+        //HTTP
+        public static string HTTPIPAddress = "http://127.0.0.1:5679/";
+        public static string HTTPTrustedIPAddress = "127.0.0.1:5679";
```

Comment:
what is this HTTPTrustedIPAddress used for? It has no implementation from what i can see.